src/layers/group.rs

```diff
@@ -78,11 +78,15 @@ impl GroupLayerData {
     }
 }
 
-map_wrapper!(GroupLayer => GroupLayerData);
+map_wrapper!(
+    #[doc = "A group layer, used to organize the layers of the map in a hierarchy."]
+    #[doc = "## Note"]
+    #[doc = "This type is currently nonstandard, as its attributes do not recursively affect child layers."]
```

Comment:
I think they shouldn't, and that this is up to the library or application implementing the rendering.

Also, suppose you'd render a Tiled map by replicating it in an entity hierarchy. Then, child entities (layers or objects) could naturally inherit transformations and visibility from their parent entities. Doing it in this library as well in such a case would only make things more complicated.

In Tiled, I do have a few helper functions that are used where needed, like `Layer::effectiveOpacity`, `Layer::effectiveParallaxFactor`, etc.

Reply:
Then I think the documentation should be updated as well. Right now it reads as if any implementation must ensure that the children also have the properties of the parent: https://doc.mapeditor.org/en/stable/reference/tmx-map-format/#group
Specifying "in the editor" would be helpful.

Reply:
Hmm, maybe, though that makes it sound strangely specific to the editor, as if it would not be the expected behavior elsewhere. The documentation is trying to communicate the expected behavior, but I don't think it needs to specify whether that is implemented in the loader, the renderer or the application. It's our choice what to do in `rs-tiled`, and I think we should leave that to the user of the library.